### PR TITLE
Enabling Change Lists

### PR DESCRIPTION
**Pull request by codyhoag**
@danielkocsis Can you please review the Change Lists intro and enablement articles? They're fairly basic at this point, so please let me know any additional details that should be covered.

I'll likely be sending these for review piecemeal, since I'm trying to squeeze these out in between other documentation emergencies. ✍️ 

Also, could you clone the `liferay/liferay-docs` repo? I have a feeling there will be more doc reviews coming your way. 😉 Thanks!

### CODE REVIEW

File: user/articles/170-content-publication-management/03-change-lists/01-intro.markdown [hunk 1]
```diff
@@ -1,9 +1,26 @@
...
+- Asset addition, edit, or deletion
+- Asset display (e.g., page, layouts, widget preferences)
+
+| **Note:** Change lists support for pages and layouts are still in development
+| and not available for @product@ 7.2 GA1.
+
+The following content types are supported by change lists at this time:
+
+- Web Content
+- DDM Template
```
Comment (danielkocsis) [inline user/articles/170-content-publication-management/03-change-lists/01-intro.markdown]: I would use Web Content Template and Structure here as DDM is more like a framework and many other assets like DL and DDL and Forms use it, but currently we limited the scope to Journal on purpose.

File: user/articles/170-content-publication-management/03-change-lists/02-enabling-change-lists.markdown [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+![Figure 1: You can enable Change Lists from the Control Panel.](../../../images/enabling-change-lists.png)
+
+3.  Save your selection to enable the Change Lists functionality.
+
+    The *Save* button saves your selection. Choosing the *Save and Go to
+    Overview* option saves your selection and also navigates you to the Change
+    Lists overview page.
+
+Your changes are now tracked globally! The current environment will serve as
+your first change list (baseline) until it's published. All
```
Comment (danielkocsis) [inline user/articles/170-content-publication-management/03-change-lists/02-enabling-change-lists.markdown]: This is a bit confusing for me. When you turn on the feature, the only change list you have is the production, which is really just a virtual change list as it represents the environment everyone sees currently. You need to create a change set manually to start editing the supported content without showing it to your audience immediately.

**Comment by danielkocsis** [comment]
Hi @codyhoag,

I added a few comments to the pages.

File: user/articles/170-content-publication-management/03-change-lists/02-enabling-change-lists.markdown (hunk 2 above)
Comment (codyhoag) [reply user/articles/170-content-publication-management/03-change-lists/02-enabling-change-lists.markdown]: Good catch; I see what you mean. I'll explain that the production environment is the initial setting and that they should create a new change list to begin leveraging the framework.

**Comment by codyhoag** [comment]
Will send another PR when I have more docs. Thanks!

**Pull request closed without merging**